src/orb.yml.hbs

```diff
@@ -564,17 +570,20 @@ commands:
                     --cluster << parameters.cluster-name >> \
                     --services ${SERVICE_NAME} \
                     --output text \
-                    --query 'services[0].deployments[].[taskDefinition, status]')
+                    --query 'services[0].deployments[].[taskDefinition, status]' \
+                    --profile << parameters.profile-name >>)
```

Comment:
I don't think this will work as the `profile-name` parameter should also be defined in the command and the job should pass the parameter to the command

Reply:
aws-cli-orb creates the default profile. Couldn't we follow the same pattern ? @lokst can you explain your concern about having a profile name empty please ?

Reply:
@sylwit I'd like to avoid the chance of unintended effects on users as the orb's commands may not necessarily be used with the aws-cli orb. Hence, I'd rather `--profile` not be set if the user did not specify a non-empty value.